.github/workflows/test-component-starter.yml

```diff
@@ -0,0 +1,68 @@
+name: Component Starter Smoke Test
+
+on:
+  workflow_call:
+    # Make this a reusable workflow, no value needed
+    # https://docs.github.com/en/actions/using-workflows/reusing-workflows
+
+jobs:
+  analysis_test:
+    name: (${{ matrix.os }}.${{ matrix.node }})
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ['14', '16']
+        os: ['ubuntu-latest', 'windows-latest']
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Get Core Dependencies
+        uses: ./.github/workflows/actions/get-core-dependencies
+
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+
+      - name: Download Build Archive
+        uses: ./.github/workflows/actions/download-archive
+        with:
+          name: stencil-core
+          path: .
+          filename: stencil-core-build.zip
+
+      - name: Pack the Build Archive
+        run: npm pack
+        shell: bash
+
+      - name: Move the Stencil Build Artifact
+        # there isn't a great way to get the output of `npm pack`, just grab the most recent and hope for the best
+        run: mv $(ls -t *.tgz | head -1) stencil-eval.tgz
+        shell: bash
+
+      - name: Initialize Component Starter
+        run: npm init stencil component tmp-component-starter
+        shell: bash
+
+      - name: Install Component Starter Dependencies
+        run: npm install
+        working-directory: ./tmp-component-starter
+        shell: bash
+
+      - name: Install Stencil Eval
+        run: npm i ../stencil-eval.tgz
```

Comment:
although, would we have wildcard expansion here? if so, could we do `npm i ../*.tgz` ?

Reply:
I'm not sure I understand - wouldn't changing this to `../*.tgz` introduce wildcard expansion?

Reply:
There should, theoretically, only ever be one build artifact, right? So this _should_ work. I don't really have a preference either way

Reply:
should have phrased my question better! Was wondering if the wildcard in `npm i ../*.tgz` would be expanded properly, if the command is running in `bash` I would think so but I don't think I've tried that before in gh actions. In any case, I think this can be merged as-is and we can fiddle with that later if we want

Reply:
Works for me!